.github/workflows/coverage.yml

```diff
@@ -28,7 +29,7 @@ jobs:
 
       - name: Run coverage
         shell: bash
-        run: forge coverage --report summary --report lcov
+        run: forge coverage --report summary --report lcov --match-contract Unit
```

Comment:
- when writing `.tree` files, the `Unit` part of the module under test is usually skipped (at least it is on this repo)
- then, if testcases are generated with `bulloak scaffold`, the contract names won't include `Unit`
- therefore, those tests won't be run, and we'll have false CI negatives and lack of coverage

I'd pass `--match-path test/unit/*` instead (remember match path uses globs instead of regex)